### ISSUE TITLE
Hostage RP Hindered By Unclear Rule

### ISSUE THREAD

**Issue by rosieposieeee** [DeltaV-Station/Delta-v#1082]
## Description

![image](https://github.com/DeltaV-Station/Delta-v/assets/52761126/cc171936-11d2-4324-90bd-9376e3e9bc2e)

The wording here is not explicit enough and leaves too much room for interpretation. 'Some leeway' is way too vague. I think it would be best to add clarifications, like:
- Heads _can_ give up objective items under threat.
- Antags, including Nuclear Operatives, _can_ take hostages instead of just killing everyone.
- Clarification on whether or not they can give up the nuke disk specifically, since it seems like it might be different than, say, the Ian photobook.

It might also be good to have a rule against ruining such roleplay scenarios for Sec/Command in section 13, such as:

**"Do not interrupt or disrupt hostage or antagonist roleplay by single-handedly ruining it for everyone. If someone says they have a bomb, or has taken someone hostage, you should not run in and get them to blow themselves up just because reviving everyone is easy."**

**Comment by Axionyxx** [comment]
Because this is unrelated to the codebase, this isn't an issue that should be on the repository. I'm not 100% sure what the correct place for it is, but I've informed Fireheart regardless.

I'd assume [Forum Discussions](https://discord.com/channels/968983104247185448/1019655026987237387) on the [Discord](https://go.delta-v.org/AtDxv) server is the best place for it, but I'm not too sure.

**Comment by Squishy77** [comment]
I kinda created the nukie hostage rp idea. The main thing I did was try to host the hostages in space near the ship. I generally think it's funny and a bit of a skill issue if the hostage situation gets interrupted by crew members or Rouge secoffs. Which happened that round but we were able to route them out before they killed -both hostages( they killed one and got killed themselves) . During another round however once a fight broke out the hostages were killed even as the nukies stated they had acid implants in them, and held them infront of them. Which is more on the chaos part, I think  the current rule set is fine
